net/rpc/server_test.go

```diff
@@ -490,7 +490,7 @@ func TestServeRequestWithInterceptor(t *testing.T) {
 		}
 
 		// let the RPC req happen
-		handler()
+		_ = handler()
```

Comment:
Could we make sure that there's no error in the happy path?

```suggestion
		err := handler()
		if err != nil {
			t.Errorf("expected handler err to be nil. Was %s", err)
		}
```